### PR TITLE
GEOPY-746

### PR DESCRIPTION
**Pull request by domfournier**
**GEOPY-746 - Ground TEM (large-loop)**

### CODE REVIEW

File: geoh5py/objects/surveys/electromagnetics/base.py [hunk 1]
```diff
@@ -477,3 +480,110 @@ def unit(self, value: str):
...
+        "Milliseconds (ms)",
+        "Microseconds (us)",
+        "Nanoseconds (ns)",
+    ]
+
+    __INPUT_TYPE = ["Rx", "Tx", "Tx and Rx"]
+
+    @property
+    def default_input_types(self) -> list[str]:
+        """Input types. Must be 'Rx only'"""
```
[the remark this answers is not in the document]
Comment (domfournier) [reply geoh5py/objects/surveys/electromagnetics/base.py:498]: I like the idea, but realized that we should make this happen further down the sub classes.

[the remark this answers is not in the document]
Comment (domfournier) [reply geoh5py/objects/surveys/electromagnetics/base.py:498]: interesting ... pylint rejects it
image

File: geoh5py/objects/surveys/electromagnetics/base.py [hunk 2]
```diff
@@ -477,3 +480,110 @@ def unit(self, value: str):
...
+        return self.__INPUT_TYPE
+
+    @property
+    def default_units(self) -> list[str]:
+        """Accepted time units. Must be one of "Seconds (s)",
+        "Milliseconds (ms)", "Microseconds (us)" or "Nanoseconds (ns)"
```
[the remark this answers is not in the document]
Comment (domfournier) [reply geoh5py/objects/surveys/electromagnetics/base.py:504]: Wish but can't

**Review by benk-mira — DISMISSED** [review]
Looks good

**Pull request merged**